lib/fly-ruby/railtie.rb

```diff
@@ -1,19 +1,7 @@
 # frozen_string_literal: true
 
 class Fly::Railtie < Rails::Railtie
-  def hijack_database_connection
-    ActiveSupport::Reloader.to_prepare do
-      # If we already have a database connection when this initializer runs,
-      # we should reconnect to the region-local database. This may need some additional
-      # hooks for forking servers to work correctly.
-      if defined?(ActiveRecord)
-        config = ActiveRecord::Base.connection_db_config.configuration_hash
-        ActiveRecord::Base.establish_connection(config.symbolize_keys.merge(Fly.configuration.regional_database_config))
```

Comment:
We're no longer using `Fly.configuration.regional_database_config`.  Is it considered part of the public API?  If not, we can remove it.

Reply:
Can remove